README.md

```diff
@@ -1 +1,23 @@
-# Microverse-portfolio
\ No newline at end of file
+# Setup-and-mobile-version-skeleton
+
+> This is my portfolio Microverse project
+Additional description about the project and its features.
+
+## Built With
+
+- html
+- css
+- github
+
```

Comment:
- Please, kindly add the `Getting started` headers by telling the user how to have your project running locally on their machine.  Example:

## Getting started

To get a local copy up and running follow these simple example steps.

1. How to clone the repo, how to move into the cloned repo, 
2. Prerequisite: e.g `vscode editor`, a `web browser(chrome, etc)`.

Reply:
- Like the previous reviewer has earlier stated, kindly update the getting started section of your readme so as to enable anyone checking your project to have a better understanding of how to use or navigate through your project.